### PR TITLE
Update npc_enchanter.cpp

### PR DESCRIPTION
**Pull request by poszer**
Changed style. (Removed low level enchants)

Preview

### CODE REVIEW

File: src/npc_enchanter.cpp [hunk 1]
```diff
@@ -267,24 +267,24 @@ class npc_enchantment : public CreatureScript
...
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Shield]", GOSSIP_SENDER_MAIN, 3);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Head]", GOSSIP_SENDER_MAIN, 4);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Shoulders]", GOSSIP_SENDER_MAIN, 5);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Cloak-]", GOSSIP_SENDER_MAIN, 6);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Chest]", GOSSIP_SENDER_MAIN, 7);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Bracers]", GOSSIP_SENDER_MAIN, 8);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Gloves]", GOSSIP_SENDER_MAIN, 9);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Legs]", GOSSIP_SENDER_MAIN, 10);
-                player->ADD_GOSSIP_ITEM(1, "[Enchant Feet]", GOSSIP_SENDER_MAIN, 11);
+                player->ADD_GOSSIP_ITEM(1, "|TInterface/ICONS/trade_engraving:24:24:-18|t[Welcome to the enchanting NPC!]", GOSSIP_SENDER_MAIN, 0);
```
Comment (BarbzYHOOL) [inline src/npc_enchanter.cpp]: Why [ ]?

Comment (poszer) [reply src/npc_enchanter.cpp]: It was ugly to leave it without icon on top, so i decide to put Enchanting Icon on it.
Also for that ''Succesfully Enchant text color" replaced red color with modern one, bcoz with red it  looks llike Warning :rofl: 
Preview

**Comment by poszer** [comment]
> 
> 
> good improvement

Thank you! @BarbzYHOOL

File: src/npc_enchanter.cpp (hunk 1 above)
Comment (BarbzYHOOL) [reply src/npc_enchanter.cpp]: I mean, why the "Welcome" is in [ ]

Comment (poszer) [reply src/npc_enchanter.cpp]: Oh, i'll edit it :+1:

Comment (poszer) [reply src/npc_enchanter.cpp]: Updated [3c4755f](https://github.com/azerothcore/mod-npc-enchanter/pull/1/commits/3c4755f273e0c79631e86699b6c3d7f710dd5c1d)

**Comment by BarbzYHOOL** [comment]
Btw, what do you mean by "removed low level enchants" ? (never tried the module myself)

**Comment by poszer** [comment]
> 
> 
> Btw, what do you mean by "removed low level enchants" ? (never tried the module myself)

Actually enchants who need first reputations, these 4 ( And they re obtainable from Quartermasters, when rep is farmed ) : 


Arcanum of the Flame's Soul  - Requires Kirin Tor - Honored
Arcanum of the Fleeing Shadow  - Requires Argent Crusade - Honored
Arcanum of the Frosty Soul  - Requires The Sons of Hodir - Honored
Arcanum of Toxic Warding  - Requires Knights of the Ebon Blade - Honored

**Comment by BarbzYHOOL** [comment]
and why?

**Comment by StygianTheBest** [comment]
I made very few adjustments to the Enchanter code I ported from Trinity. I do not remember the broken English - "I'm giving your service." - being in the opening paragraph. I think the whole point of this mod was to avoid requirements, so I would fork this and create a new Enchanter module before hacking this one up. :)

**Comment by BarbzYHOOL** [comment]
@StygianTheBest as you can see, nobody added the "giving your service line" so you broke the english yourself (haha, in fact it's probably from the original TC script :P but you triggered me xD)

@poszer no explanation?? I don't know, I'm just asking for what is better for the mod users

**Comment by StygianTheBest** [comment]
> @StygianTheBest as you can see, nobody added the "giving your service line" so you broke the english yourself (haha, in fact it's probably from the original TC script :P but you triggered me xD)

Sorry, but you're wrong. Look at my commit from a year ago. Triggered for nothing.. lol.

"INSERT INTO `npc_text` (`ID`, `text0_0`) VALUES (@Entry, 'Good day $N. Beauregard Boneglitter at your service. I offer a vast array of gear enchantments for the aspiring adventurer.');"

**Comment by poszer** [comment]
> 
> 
> @StygianTheBest as you can see, nobody added the "giving your service line" so you broke the english yourself (haha, in fact it's probably from the original TC script :P but you triggered me xD)
> 
> @poszer no explanation?? I don't know, I'm just asking for what is better for the mod users

Since i have edited the module from my server side, who is Instant 80 Server, i forget to restore some options there. My fault, and that's explanation :D It's better to use it here with Last update :+1:

**Comment by BarbzYHOOL** [comment]
So what about https://github.com/azerothcore/mod-npc-enchanter/pull/1/commits/1e03224f2a24cb2052bebcb047ca1efce8b077b4 what is better, to delete them or not?

**Comment by poszer** [comment]
> 
> 
> So what about [1e03224](https://github.com/azerothcore/mod-npc-enchanter/commit/1e03224f2a24cb2052bebcb047ca1efce8b077b4) what is better, to delete them or not?

Don't need to delete. I'll suggest to apply with [1e03224](https://github.com/azerothcore/mod-npc-enchanter/commit/1e03224f2a24cb2052bebcb047ca1efce8b077b4)

**Comment by BarbzYHOOL** [comment]
ok thank you @poszer :)

**Pull request merged**